package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/accordproject/markdown-editor.git"
   },
   "scripts": {
-    "test": "jest",
+    "test": "jest --passWithNoTests",
```

Comment:
Is this temporary until new tests are created?

Reply:
Yes -- and it only seems to be needed for the Travis build. Not sure why.